### DISCUSSION TITLE
【Keyball44】LEDのランドが剥離してしまいました。

### DISCUSSION THREAD

**Discussion by helosshi** [Yowkees/keyball#294] [Q&A]
20番のLEDがつかず、ハンダを除去する際に、ランドも剥離してしまいました。
以下のどちらかを実現できると助かります。

1. 20番も復活させる方法
2. 20番を飛ばし、21番から復活させる方法

![IMG_5546](https://github.com/Yowkees/keyball/assets/4505812/89302f76-83e2-433c-a3c2-4f56db050a5e)

お忙しい中申し訳ありませんが、よろしくお願いします。

**Comment by Yowkees** [comment]
ご連絡ありがとうございます。
20番のLEDを両面テープ等で固定してから、下記画像の同色どうしをエナメル線等でショートしていただければ復活できます。
20番をスキップする場合は、19番のオレンジ色と21番の赤色を接続していただければ可能です。
以上、お手数ですがご対応お願いいたします。
![KB44](https://github.com/Yowkees/keyball/assets/84209772/51e74de0-971f-4757-9a7a-33ebcc319827)

**Reply by helosshi** [reply]
ご回答ありがとうございます！試してみます！
